tensorcircuit/quantum.py

```diff
@@ -1600,50 +1589,15 @@ def qop2tn(qop: QuOperator) -> Any:
     :return: MPO or MPS in the form of TensorNetwork
     :rtype: Union[tn.matrixproductstates.MPO, tn.matrixproductstates.MPS]
     """
-    sorted_nodes, is_mps, nwires = extract_tensors_from_qop(qop)
+    sorted_nodes, is_mps, _ = extract_tensors_from_qop(qop)
 
     tensors = []
-    for i, node in enumerate(sorted_nodes):
-        edge_to_axis = {edge: axis for axis, edge in enumerate(node.edges)}
-
-        left_v_ax, right_v_ax = None, None
-
-        if i > 0:
-            prev_node = sorted_nodes[i - 1]
-            edge_to_prev = next(
-                e for e in node.edges if (e.node1 is prev_node or e.node2 is prev_node)
-            )
-            left_v_ax = edge_to_axis[edge_to_prev]
-
-        if i < nwires - 1:
-            next_node = sorted_nodes[i + 1]
-            edge_to_next = next(
-                e for e in node.edges if (e.node1 is next_node or e.node2 is next_node)
-            )
-            right_v_ax = edge_to_axis[edge_to_next]
-
+    for _, node in enumerate(sorted_nodes):
         if is_mps:
             tensors.append(node.tensor)
 
         else:  # MPO
-            left_v_ax, right_v_ax, phys_in_ax, phys_out_ax = 0, 1, 2, 3
-            if node.tensor.ndim == 3:
-                if i == 0:
-                    right_v_ax, phys_in_ax, phys_out_ax = 0, 1, 2
-                    permuted_tensor = backend.transpose(
-                        node.tensor, perm=[right_v_ax, phys_in_ax, phys_out_ax]
-                    )
-                else:
-                    left_v_ax, phys_in_ax, phys_out_ax = 0, 1, 2
-                    permuted_tensor = backend.transpose(
-                        node.tensor, perm=[left_v_ax, phys_in_ax, phys_out_ax]
-                    )
-            else:
-                permuted_tensor = backend.transpose(
-                    node.tensor, perm=[left_v_ax, right_v_ax, phys_in_ax, phys_out_ax]
-                )
-
-            tensors.append(permuted_tensor)
+            tensors.append(node.tensor)
```

Comment:
is for the logic is the same, just use `tensors = [node.tensor for node in sorted_nodes]`?